draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -139,7 +139,42 @@ Note: Issuers should be mindful of how long the usage of the refresh token is al
 
 ### Wallet Attestation Schema {#wallet-attestation-schema}
 
-[Section 3.1 of wallet attestation draft would define the basics, and this profile will define the details.]
+Wallets MUST use attestations following the definition given in [!I-D.ietf-looker-key-attestation-client-authentication].
+
+In addition, the Wallet Attestation MUST contain the following claims:
+
+* `key_type`: this claim asserts the security mechanism the wallet can use to manage private keys. This capability is based on the capabilities of the execution environent of the wallet, this might be a secure element (in case of a wallet residing on a smartphone) or a Cloud-HSM (in case of a cloud wallet). This specification defines the following values for `key_type`: `STRONGBOX`, `TEE`, `SecureEnclave`, `Software`.
+* `user_authentication`: this claim asserts the security mechanism the wallet can use to authenticate access to private keys. This specification defines the following values for `user_authentication`: `APP_PIN_6_DIGITS`, ...
+
+These additional claims inform the issuer about the security capabilities of the wallet and allows the issuer to refuse credential issuance if the achievble security level of a certain wallet does not fulfil the issuer's requirements.
```

Comment:
Do we have to defined error codes and messages for that?

Reply:
you mean in the credential issuance response?

Reply:
yes, do we have to give a ref to openid4vc?

Reply:
https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/issues/62